### PR TITLE
MRG: DOC: Extend doc of event_id and conditions

### PR DESCRIPTION
**Pull request by hoechenberger**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -467,22 +467,39 @@
 
 #  `event_id`` : dict
```
Comment (agramfort) [inline config.py:468]:
```suggestion
#  `event_id``: dict
```
no?

File: config.py [hunk 2]
```diff
@@ -467,22 +467,39 @@
...
 #    Dictionary that maps events (trigger/marker values)
-#    to conditions.
+#    to conditions. The ``/``` character can be used to group conditions.
```
Comment (agramfort) [inline config.py:470]:
```suggestion
#    to conditions. The ``/`` character can be used to group conditions.
```

File: config.py [hunk 3]
```diff
@@ -467,22 +467,39 @@
...
 # event_id = {'auditory/left': 1, 'auditory/right': 2}
 
 #  `conditions`` : dict
-#    List of condition names to consider. Must match the keys
-#    in event_id.
+#    List of condition names to consider. This can either be the keys of
+#    ``event_id``, or – if event names were specified with ``/`` for
+#    grouping – the name of the *grouped* condition (i.e., the
+#    condition name before or after that ``/`` that is shared between the
+#    respective conditions you wish to group).
```
Comment (agramfort) [inline config.py:493]: you can point to: https://mne.tools/stable/auto_tutorials/epochs/plot_10_epochs_overview.html#subselecting-epochs

File: config.py (hunk 1 above)
Comment (hoechenberger) [reply config.py:468]: I believe in numpydoc one is supposed to have the space, and that's also how it has been handled in the rest of `config.py`. I think we should keep the space for sake of consistency for now :)

File: config.py (hunk 2 above)
Comment (hoechenberger) [reply config.py:470]: Thanks, not sure if it's my keyboard layout or VS Code constantly trying to sneak in additional backticks! super annoying

File: config.py (hunk 1 above)
Comment (agramfort) [reply config.py:468]: I meant : `` -> `

File: config.py (hunk 3 above)
Comment (hoechenberger) [reply config.py]: Done

File: config.py (hunk 1 above)
Comment (hoechenberger) [reply config.py]: Oh. it's actually supposed to be two backticks on either side. Fixed. sorry about that.

File: config.py [hunk 4]
```diff
@@ -465,24 +465,44 @@
...
 
 #  `conditions`` : dict
-#    List of condition names to consider. Must match the keys
-#    in event_id.
+#    List of condition names to consider. This can either be the keys of
+#    ``event_id``, or – if event names were specified with ``/`` for
+#    grouping – the name of the *grouped* condition (i.e., the
+#    condition name before or after that ``/`` that is shared between the
+#    respective conditions you wish to group). See the "Subselecting epochs"
+#    tutorial for more information: https://mne.tools/stable/auto_tutorials/epochs/plot_10_epochs_overview.html#subselecting-epochs  # noqa: 501
```
Comment (jasmainak) [inline config.py:496]: Do you explain what `None` does?

Comment (hoechenberger) [reply config.py:496]: No, it was not documented before and I have no idea what it does :D Can you enlighten me? Then I'd be happy to add it to the description!

Comment (jasmainak) [reply config.py:496]: I think `None` will consider all epochs instead of subselecting some

Comment (hoechenberger) [reply config.py:496]: Wow that’s quite a stretch then, semantically... None means all. We should reconsider this (in a separate PR)

Comment (hoechenberger) [reply config.py:513]: So I've checked the code and it appears to me that `conditions = [None]` shouldn't really work. So for now, I have removed it from the example `config.py`: 25ce6a0

File: config.py [hunk 5]
```diff
@@ -482,30 +482,49 @@
 # XXX not needed if bids events are present
 # min_event_duration = 0.002
 
-#  `event_id`` : dict
+#  ``event_id`` : dict
 #    Dictionary that maps events (trigger/marker values)
-#    to conditions.
+#    to conditions. The ``/``` character can be used to group conditions.
```
Comment (agramfort) [inline config.py:487]:
```suggestion
#    to conditions. The ``/`` character can be used to group conditions.
```

Comment (hoechenberger) [reply config.py:487]: AGAIN one of those!!! Really need to get this sorted out… Thanks for catching this!

**Comment by hoechenberger** [comment]
@jasmainak Let me know it it's okay with you to merge (or directly merge yourself). Regarding the `conditions = [None]` thing, I believe we should discuss which behavior we want exactly, and how to expose the functionality to the user.

**Pull request merged**